packages/atlas/README.md

```diff
@@ -35,7 +36,7 @@ class MyApp extends StatelessWidget {
 
 class AtlasSample extends StatelessWidget {
   final CameraPosition _initialCameraPosition = CameraPosition(
```

Comment:
Unrelated to MapMarkers but I think we should discuss what we think we need for CameraPosition.  I think target and zoom are sufficient for now.